src/insert-style.js

```diff
@@ -1,9 +1,9 @@
 export function getStyle(elm) {
```

Comment:
Are you referring to flipping the IE8 detection conditionals? I put a note about it in the commit message but I've been rewriting history here like crazy for the PR so it's easy to miss. :) 

The 'textContent' property of a jsdom-created style element always seems to be
present (even after attempting to delete), so testing the IE8 case was proving
difficult. Inverting the conditional makes it much easier, as we're now able to
pass in a phony IE8 style element by pre-populating styleSheet.cssText.